DS_and_Algorithm/Day3/Selectionsort.py

```diff
@@ -0,0 +1,22 @@
+def selection_sort(arr):
+    n = len(arr)
+
+    for i in range(n):
+        # Find the minimum element in the unsorted part
+        min_index = i
+        for j in range(i-1, n):
+            if arr[j] > arr[min_index]:
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Incorrect comparison operator for sorting</b></div>
<div id="fix">

The comparison operator `>` will sort in descending order. For ascending order sort, consider using `<` operator.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
            if arr[j] < arr[min_index]:
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/35#issuecomment-2535997439>#5fbf54</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged